src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -25,15 +25,24 @@ public class Metric
         internal static readonly double[] DefaultHistogramBounds = new double[] { 0, 5, 10, 25, 50, 75, 100, 250, 500, 1000 };
         private AggregatorStore aggStore;
 
-        internal Metric(Instrument instrument, AggregationTemporality temporality, string metricName = "")
-            : this(instrument, temporality, DefaultHistogramBounds, metricName)
+        internal Metric(
+            Instrument instrument,
+            AggregationTemporality temporality,
+            string metricName,
+            string metricDescription)
+            : this(instrument, temporality, DefaultHistogramBounds, metricName, metricDescription)
         {
         }
 
-        internal Metric(Instrument instrument, AggregationTemporality temporality, double[] histogramBounds, string metricName = "")
+        internal Metric(
+            Instrument instrument,
+            AggregationTemporality temporality,
+            double[] histogramBounds,
```

Comment:
Move `histogramBounds` to the end make its default value as `DefaultHistogramBounds`? That way we wouldn't need the other ctor?

Reply:
i tried that, but then realized the default must be a const, and arrays cannot be used for that purpose!

Reply:
Have the default value of `histogramBounds` as `null` and then assign that as `histogramBounds` inside the method?